src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,38 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+import { Tab } from '../../types/Tab';
+
+type Props = {
+  tabs: Tab[],
+  selectedTab: Tab,
+  onTabSelected: (tab: Tab) => void;
+};
+
+export const Tabs: React.FC<Props> = ({ tabs, selectedTab, onTabSelected }) => (
+  <div>
+    <div className="tabs is-boxed">
+      <ul>
+        {tabs.map(tab => (
+          <li
+            key={tab.id}
+            className={classNames({ 'is-active': tab.id === selectedTab.id })}
+          >
+            <a
+              href={`#${tab.id}`}
+              onClick={() => (
+                selectedTab.id !== tab.id && (
+                  onTabSelected(tab)
+                )
+              )}
```

Comment:
you already have this logic in parent handler
```suggestion
                onTabSelected(tab)
              )}
```